src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -5808,4 +5808,32 @@ elementFormDefault="qualified">
     </xs:complexType>
   </xs:element>
 
+  <xs:element name="InternalsVisibleTo" substitutionGroup="msb:Item">
+  <xs:annotation>
+    <xs:documentation>
+      <!-- _locID_text="InternalsVisibleTo" _locComment="" -->Specifies that types that are ordinarily visible only within the assembly are visible to the specified assemblies.
+    </xs:documentation>
+  </xs:annotation>
+  <xs:complexType>
+      <xs:complexContent>
+        <xs:extension base="msb:SimpleItemType">
+          <xs:attribute name="Include" type="xs:string">
+            <xs:annotation>
+              <xs:documentation>
+                <!-- _locID_text="InternalsVisibleTo_Include" _locComment="" -->The name of the friend assembly to make internal types visible to, e.g. Microsoft.AspNetCore.
```

Comment:
```suggestion
                <!-- _locID_text="InternalsVisibleTo_Include" _locComment="" -->The name of the friend assembly that gains visibility to internals of the other assembly.
```
?

Reply:
The text is basically lifted exactly from [the ref docs for the attribute](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=net-5.0). I think we should keep them the same so if we want to change it here we should change it there too.

__`InternalsVisibleToAttribute` class description__:
Specifies that types that are ordinarily visible only within the current assembly are visible to a specified assembly.

__`AssemblyName` property description__:
Gets the name of the friend assembly to which all types and type members that are marked with the internal keyword are to be made visible.

Reply:
Personally, I would prefer they both change. It had been very unclear, and it is in our style docs not to use things like `e.g.` because they aren't technically English.

If you want to leave it this way, though, I'm not blocking its insertion.